### PR TITLE
[fix] fix bugs in AVS precompile

### PR DESCRIPTION
**Pull request by trestinlsd**
## Description
fix bugs in AVS precompile

### CODE REVIEW

File: precompiles/avs/types.go [hunk 1]
```diff
@@ -41,8 +43,8 @@ func (p Precompile) GetAVSParamsFromInputs(_ sdk.Context, args []interface{}) (*
 	if !ok || avsOwnerAddress == "" {
 		return nil, xerrors.Errorf(exocmn.ErrContractInputParaOrType, 4, "string", avsOwnerAddress)
 	}
-	avsParams.AvsOwnerAddress = avsOwnerAddress
-
+	AvsOwnerAddressHex, _ := util.DecodeHexString(avsOwnerAddress)
```
Comment (mikebraver) [inline precompiles/avs/types.go]: AvsOwnerAddressHex => avsOwnerAddressHex

File: precompiles/avsTask/methods.go [hunk 2]
```diff
@@ -31,14 +31,7 @@ func (p Precompile) RegisterAVSTask(
...
 	callerAddress, _ := bech32.EncodeFromBase256("exo", contract.CallerAddress.Bytes())
-	avs, err := p.avsKeeper.GetAVSInfo(ctx, callerAddress)
-	if err != nil {
-		return nil, err
-	}
-
-	if avs.GetInfo() == nil {
-		return nil, fmt.Errorf(ErrNotYetRegistered, contract.CallerAddress)
-	}
+	_, err := p.avsKeeper.GetAVSInfo(ctx, callerAddress)
```
Comment (mikebraver) [inline precompiles/avsTask/methods.go]: Why disable the error checking? If the operator address is not exist, won't it return error? If for test purpose, we need to register the operator firstly before this task validate, right?

File: utils/utils.go [hunk 3]
```diff
@@ -86,3 +87,12 @@ func GetExocoreAddressFromBech32(address string) (sdk.AccAddress, error) {
 
 	return sdk.AccAddress(addressBz), nil
 }
+func DecodeHexString(hexString string) ([]byte, error) {
+	if strings.HasPrefix(hexString, "0x") || strings.HasPrefix(hexString, "0X") {
+		hexString = hexString[2:]
+	}
+	if len(hexString)%2 != 0 {
```
Comment (mikebraver) [inline utils/utils.go:97]: please make lint locally

File: precompiles/avsTask/methods.go (hunk 2 above)
Comment (trestinlsd) [reply precompiles/avsTask/methods.go]: fixed

File: precompiles/avs/types.go (hunk 1 above)
Comment (trestinlsd) [reply precompiles/avs/types.go]: fixed

**Review by mikebraver — APPROVED** [review]

**Pull request merged**